### ISSUE TITLE
Is it possible to get the default pitch and yaw of a scene that's not the current one?

### ISSUE THREAD

**Issue by Doniee** [mpetroff/pannellum#438]
Hey,
I've made a tour with hotspots and also created a map that enables jumping to a specific scene (using loadScene). I was wondering if I can somehow open a scene with the same pitch and yaw settings that are used if the hotspots are clicked? I have defined the default settings when I create the scenes, but can I access the values somehow? Something like the getConfig function, but for any scene, not just the current one. Or even the whole configuration, not just for one scene. The loadScene seems to accept pitch and yaw as parameters, but I would like to get them from somewhere dynamically. 

Thanks!

**Comment by mpetroff** [comment]
Have you tried just looking at the JSON configuration that you're using directly?

**Comment by Doniee** [comment]
I tried, just couldn't figure out how to access it. The documentation says that `getConfig()` returns the configuration of current scene, so I thought I couldn't get the whole thing from there. But now I noticed that `viewer.getConfig()` also returns the scenes, so I can get the values from there. Thanks!

(By the way, I also have a problem with stopAutoRotate, maybe a bug, but should I open a new issue for that?)
